frontend/src/Components/ReusableComponents/ReusableFunctions.js

```diff
@@ -66,6 +89,22 @@ export const emailVerification = async (token, success, error, info, setparams,
    setparams("");
 };
 
+export const onUserLogout = async (accessToken, localStorageItem, stateToChange) => {
+   try {
+      await AxiosInstance.post(
+         `/auth/logout`,
+         {},
+         {
+            headers: { Authorization: accessToken },
+         }
+      );
+      localStorage.removeItem(localStorageItem);
+      stateToChange("");
+   } catch (err) {
+      console.log(err);
```

Comment:
You can use also console.err but always better to have some more user friendly error handling.

Reply:
Fixed.